draft-ietf-quic-ack-frequency.md

```diff
@@ -251,25 +251,25 @@ Request Max Ack Delay:
 
 Reordering Threshold:
 
-: A variable-length integer that indicates how many
+: A variable-length integer that indicates how
```

Comment:
I think we need the "many" here, no?

Reply:
It's not about the number of packets, but the amount of reordering.  There's probably a clearer way to say that, though?

Reply:
right... Some like "indicates the maximum packet number gap before eliciting an immediate ACK"

Reply:
Thanks, that's good.  The word cap could be confusing, so I went for "maximum amount of packet reordering before eliciting an immediate ACK."  I think that's conceptually correct?